packages/editor/api-report.md

```diff
@@ -355,7 +366,7 @@ export class App extends EventEmitter {
         description: string;
     }>;
     get onlySelectedShape(): TLBaseShape<any, any> | null;
-    openMenus: Set<string>;
```

Comment:
This makes this PR a `major` breaking change, you currently have it marked as a `patch`